wp-content/themes/missilethreat/assets/_scss/components/_tables.scss

```diff
@@ -0,0 +1,59 @@
+@use 'abstracts' as *;
+
+table {
```

Comment:
It might be worth putting this inside of the .content class, just in case any plugins or such use tables in their layout so they don't conflict.